client/tailscale/apitype/apitype.go

```diff
@@ -10,6 +10,7 @@ import "tailscale.com/tailcfg"
 const LocalAPIHost = "local-tailscaled.sock"
 
 // WhoIsResponse is the JSON type returned by tailscaled debug server's /whois?ip=$IP handler.
+// In successful whois responses, Node and UserProfile are never nil.
 type WhoIsResponse struct {
 	Node        *tailcfg.Node
```

Comment:
While here might as well add docs about which Node and which UserProfile this refers to. And can then say always non-nil in their respective doc strings

Reply:
I don't know how to describe those fields, in particular I don't know what UserProfile represents for tagged devices. Sounds like you do though, so please send a followup?

Reply:
Will do